packages/framework/fluid-framework/src/index.ts

```diff
@@ -19,31 +19,56 @@ export { ConnectionState } from "@fluidframework/container-loader";
 export type {
 	ContainerAttachProps,
 	ContainerSchema,
-	DataObjectClass,
 	IConnection,
 	IFluidContainer,
 	IFluidContainerEvents,
 	IMember,
 	InitialObjects,
 	IServiceAudience,
 	IServiceAudienceEvents,
-	LoadableObjectClass,
-	LoadableObjectClassRecord,
 	MemberChangedListener,
 	Myself,
 } from "@fluidframework/fluid-static";
+export type { SharedObjectKind, ISharedObjectEvents } from "@fluidframework/shared-object-base";
 
 // Let the tree package manage its own API surface, we will simply reflect it here.
 // Note: this only surfaces the `@public` API items from the tree package. If the `@beta` and `@alpha` items are
 // desired, they can be added by re-exporting from one of the package's aliased export paths instead (e.g. `tree
 // alpha` to surface everything `@alpha` and higher).
-// eslint-disable-next-line no-restricted-syntax
+// eslint-disable-next-line no-restricted-syntax, import/export
 export * from "@fluidframework/tree";
 
+import type { SharedObjectKind } from "@fluidframework/shared-object-base";
+import { SharedTree as OriginalSharedTree } from "@fluidframework/tree/internal";
+import { type ITree } from "@fluidframework/tree";
+
+/**
+ * A hierarchical data structure for collaboratively editing strongly typed JSON-like trees
+ * of objects, arrays, and other data types.
+ * @public
+ */
+// eslint-disable-next-line import/export
+export const SharedTree: SharedObjectKind<ITree> = OriginalSharedTree;
```

Comment:
Here we reexport SharedTree, but with the alpha types (ISharedObjectKind) removed, just keeping the SharedObjectKind.

Reply:
Should any of your PR comments make it into code comments?

Reply:
I have improved the docs comments a bit. Let me know if there is anything you spot thats still missing.

Reply:
Oops, I missed this file, I'll improve its docs in a moment

Reply:
Should be better now.